bootstrap/api/requests_test.go

```diff
@@ -13,41 +13,67 @@ import (
 )
 
 func TestAddReqValidation(t *testing.T) {
+	const (
```

Comment:
Define this at the top of the file and use `testsutil.GenerateUUID(&testing.T{})` to generate UUIDs